react-native/react/tracker/index.js

```diff
@@ -36,6 +37,18 @@ type TrackerProps = {
 class Tracker extends Component {
   props: TrackerProps;
 
+  componentWillMount () {
+    this.onemin = setInterval(function () {
```

Comment:
onemin -> checkTrackingTimer
also we should check to see if it exists in case of componentWillMount being called multiple times or unmount not being called